modules.d/06dbus/module-setup.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# called by dracut
+check() {
+  require_binaries dbus-daemon || return 1
+
+  return 255
+}
+
+depends() {
+  echo systemd
+  return 0
+}
+
+install() {
+  local DBUS_SERVICE=/usr/lib/systemd/system/dbus.service
+  if [[ -e $DBUS_SERVICE ]]; then
+    if [[ -L $DBUS_SERVICE ]]; then
+      DBUS_SERVICE=$(readlink $DBUS_SERVICE)
+    fi
+  else
+    DBUS_SERVICE=/etc/systemd/system/dbus.service
+    if [[ -e $DBUS_SERVICE ]]; then
+      if [[ -L $DBUS_SERVICE ]]; then
+        DBUS_SERVICE=$(readlink $DBUS_SERVICE)
+      fi
+    else
+      echo "Could not find dbus.service";
+      exit 1
+    fi
+  fi
+
+  inst_multiple \
+    $DBUS_SERVICE \
+    /usr/lib/systemd/system/dbus.socket \
+    /usr/bin/dbus-daemon \
+    /usr/bin/dbus-send
```

Comment:
Maybe install `busctl` instead? Nowadays I find it more useful.

Reply:
Done